applications/ecallmgr/src/ecallmgr_fs_conference.erl

```diff
@@ -234,11 +248,14 @@ publish_event(Action, Props, Node) ->
         {'error', 'not_found'} -> lager:debug("not publishing conference event ~s for not existant ~s ", [Action, UUID])
     end.
 
-publish_event(Action, #conference{handling_locally='true'} = Conference, Props, _Node) ->
-    Event = conference_event(Action, Conference, Props),
-    publish_event(Event);
-publish_event(Action, _Conference, _Props, _Node) ->
-    lager:debug("conference control on another node, not publishing event ~s", [Action]).
+publish_event(Action, #conference{handling_locally=IsLocal} = Conference, Props, _Node) ->
+    case props:is_true(<<"Force-Publish-Event-State">>, Props, 'false')
+        orelse props:is_true(<<"Publish-Event-State">>, Props, 'true')
+        andalso IsLocal
```

Comment:
I understand this `andalso`'s left hand side only encloses `props:is_true(<<"Publish-Event-State">>, Props, 'true')`.
The `Force...` expression is thus the top-level expression: `A orelse B andalso C` has this precedence: `A orelse (B andalso C)`.

Is this expected? Should we add braces for clarity?